package.json

```diff
@@ -27,5 +25,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "devDependencies": {}
```

Comment:
### [IMPORTANT CHANGE]
We do have a `eslintrc.js` file but we don't seem to have `eslint` and `eslint` related packages installed. Let's fix this. You may run

```
npx eslint --init
```
I highly recommend you use the AirBnB style guide

Reply:
I have to install it a couple of times but it gives an error when I want to start my app and all recommended fixes are not fixing it.